int8 count
float32[] back_legs
float32[] front_legs